src/plugins/intel_gpu/src/kernel_selector/core/actual_kernels/activation/activation_kernel_base.cpp

```diff
@@ -28,7 +28,7 @@ ActivationKernelBase::DispatchData ActivationKernelBase::SetDefault(const activa
         dispatchData.gws = {out.X().v * out.Y().v, Align(out.Feature().v, 16), Align(out.Batch().v, 16)};
         dispatchData.lws = {1, 16, 16};
     } else {
-        dispatchData.gws = {out.X().v, out.Y().v * out.Z().v, out.Feature().v * out.Batch().v};
+        dispatchData.gws = {out.X().v*out.W().v, out.Y().v * out.Z().v, out.Feature().v * out.Batch().v};
```

Comment:
please add a blank before and after the asterisk. (why isn't it caught during cpplint?)

Reply:
Fixed.